packages/@uppy/tus/src/index.js

```diff
@@ -216,7 +216,7 @@ module.exports = class Tus extends BasePlugin {
           opts.onBeforeRequest(req)
         }
 
-        if (Object.hasOwn(queuedRequest, 'shouldBeRequeued')) {
+        if (queuedRequest.hasOwnProperty('shouldBeRequeued')) {
```

Comment:
that's a very good point. maybe like this?
```suggestion
        if (queuedRequest != null && queuedRequest.hasOwnProperty('shouldBeRequeued')) {
```

Reply:
I don't think strict equal is a good idea in this case, because if queuedRequest is `undefined` (which it will be in this case), then `queuedRequest !== null` will be true, so it would evaluate `queuedRequest.hasOwnProperty` and crash. I think `?.` is the most elegant solution yes :)

Reply:
I think most commonly lose equality is considered a bad practice, if it could be that then I think `typeof queuedRequest === 'object' && queuedRequest !== null` is safer

Reply:
looks like there's a util for this anyway though :)